e2e/isolated/isolated_suite_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package isolated_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/aws/copilot-cli/e2e/internal/client"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var cli *client.CLI
+var aws *client.AWS
+var appName string
+var vpcStackName string
+var vpcStackTemplatePath string
+var vpcImport client.EnvInitRequestVPCImport
```

Comment:
move to other test file? i don't see it used here

Reply:
This seems to be how the other e2e tests have their vars, so I just followed suit(e🤣).